### DISCUSSION TITLE
MMM and OBS Websocket - Cannot change default views

### DISCUSSION THREAD

**Discussion by Supamav** [sircharlo/meeting-media-manager#1761] [Q&A] [answered]
Hello Brothers,

I'm setting up a stand alone PC for our Memorial tomorrow, I need to change the default scenes for Stage and Media as they got reversed by accident.  However I cannot change them.

Is there a way to reset or change the assigned scenes in MMM?

Thanks for your time!

Jason

**Comment by mtdvlpr** [comment, marked as answer]
@Supamav, try creating a third OBS scene and use that to switch the scenes around again. It's not ideal, so I could think of a better way for the future.

**Reply by Supamav** [reply]
Thank you very much, that was the solution.  I should have figured that one out.... :)  Thanks again for your help!  MMM is a wonderful tool!
